templates/monitoring/jobs/openshift_monitoring_federation.yaml

```diff
@@ -13,6 +13,7 @@
     - 'kubelet_volume_stats_used_bytes{endpoint="https-metrics",namespace=~"{{ index .Params "namespace-prefix" }}.*"}'
     - 'kubelet_volume_stats_available_bytes{endpoint="https-metrics",namespace=~"{{ index .Params "namespace-prefix" }}.*"}'
     - 'kubelet_volume_stats_capacity_bytes{endpoint="https-metrics",namespace=~"{{ index .Params "namespace-prefix" }}.*"}'
+    - 'haproxy_backend_http_responses_total{route=~"^keycloak.*", exported_namespace=~"redhat-rhoam.*sso"}'
```

Comment:
very slight improvement: `redhat-rhoam.*sso` -> `redhat-rhoam.*sso$` (ensure it ends in 'sso').

Reply:
Hi Peter, thank you very much for review and comments.
Fix done,  + new Alerts changed to support RHMI, so the change was:
redhat-rhoam.*sso -> redhat-.*sso$
Testing done, so hope it's ok.